src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const oneDayCost = 40;
+  const threeDays = 3;
+  const sevenDays = 7;
+  const minTermDiscount = 20;
+  const longTermDiscount = 50;
+  const totalPrise = days * oneDayCost;
```

Comment:
The variable name `totalPrise` is misspelled. It should be `totalPrice` to correctly represent the total cost.